kvbc/src/kvbc_adapter/replica_adapter.cpp

```diff
@@ -108,10 +109,15 @@ ReplicaBlockchain::ReplicaBlockchain(
   ConcordAssertNE(db_chkpt_, nullptr);
 }
 
-BlockId ReplicaBlockchain::deleteBlocksUntil(BlockId until) {
+BlockId ReplicaBlockchain::deleteBlocksUntil(BlockId until, bool delete_files_in_range) {
+  LOG_INFO(logger_, "deleteBlocksUntil:" << KVLOG(until));
   auto prev_dur = delete_blocks_until_duration.Get().Get();
   auto start = std::chrono::steady_clock::now();
-  auto id = deleter_->deleteBlocksUntil(until);
+  if (delete_blocks_state_transfer_cb_) {
+    delete_blocks_state_transfer_cb_(until);
```

Comment:
@glevkovich, the existing code calls this method from ```Replica::deleteBlocksUntil``` which is a wrapper to the replica adapter.
Why do we also need to add a call here? Wouldn't it be called twice?